src/mixpanelAnalytics.ts

```diff
@@ -0,0 +1,46 @@
+import Assignment from './assignment';
+
+type EventProperties = {
+  TTVisitorID: string;
+  SplitName: string;
+  SplitVariant: string | null;
+  SplitContext?: string;
+};
+
+declare global {
+  interface Window {
+    mixpanel: {
+      track(eventName: string, properties: EventProperties, callback: (value: boolean) => void): void;
+      identify(id: string): void;
+      alias(id: string): void;
+    };
+  }
+}
+
+class MixpanelAnalytics {
```

Comment:
I think we want an interface that MixpanelAnalytics conforms to. This will allow apps to introduce their own analytics client.